extensions/legit/porcelain.lisp

```diff
@@ -764,7 +764,7 @@ M	src/ext/porcelain.lisp
 (defparameter *rebase-script-content*
   #+(or lem-ncurses lem-sdl2)
   (str:from-file
-   (asdf:system-relative-pathname (asdf:find-system "lem")
+   (asdf:system-relative-pathname (asdf:find-system "lem/legit")
```

Comment:
Is it not lem-legit?

Reply:
It is! thanks.

(rebasing was working though O_o )